From 405984bf17404ad151d1a38bd3011542ea61827e Mon Sep 17 00:00:00 2001
From: Tien Le <tien.le.xw@renesas.com>
Date: Mon, 1 Mar 2021 17:45:30 +0700
Subject: [PATCH 084/135] arm64: dts: r9a07g044l: Add TSU trip point

This patch adds thermal-zone and TSU trip point for r9a07g044l SoC

Signed-off-by: Tien Le <tien.le.xw@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index 8e634d8..ef58822 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -424,6 +424,23 @@
 		};
 	};
 
+	thermal-zones {
+		sensor_thermal: sensor-thermal {
+			polling-delay-passive = <250>;
+			polling-delay = <1000>;
+			sustainable-power = <3874>;
+			thermal-sensors = <&tsu 0>;
+
+			trips {
+				sensor_crit: sensor-crit {
+				temperature = <110000>;
+				hysteresis = <1000>;
+				type = "critical";
+				};
+			};
+		};
+	};
+
 	timer {
 		compatible = "arm,armv8-timer";
 		interrupts-extended =
-- 
2.7.4

